docs/high-level-overview.md

```diff
@@ -26,14 +26,16 @@ A picture is worth a thousand words:
 
 There are four categories of Data Objects in GEMD: Materials, Processes, Measurements, and Ingredients.
 These represent real world objects in the development of materials.
-The 4 categories of Data Objects can only be linked in specific ways, for example, a Process Object can only be linked to one or many Material Objects as its input, more details are explained in Table1.
-Each Object can be represented in 3 different states, these states are defined below, they are Template, Spec, and Run.
+The four categories of Data Objects can only be linked in specific ways; for example, a Process Object can only be linked to one or many Material Objects as its input (more details are explained in Table1).
+Each Object can be represented in three different states.
```

Comment:
Is states really the right word here?  The word was unchanged, so this comment is non-blocking.